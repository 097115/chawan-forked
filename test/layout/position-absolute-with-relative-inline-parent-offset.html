<!-- don't ask -->
<p>
Note: the absolute box is sized differently than boxes in other engines,
because the CSS standard explicitly specifies this as UB and I *really*
don't want to make this even more complex.
<p>
I don't care, but feel free to fix it ;)
<p>
(Also, position: absolute should overprint inline boxes after it, but it
doesn't. This needs further investigation.)
</p>
<br>
<br>test test <span style="position:relative">test test<div style="display: inline-block">adsfasdf</div><BR>
test test test test
<div>
<div style="display: block; position: absolute; top: 0em; left: 0; height: 100%; background: red; width: 3em">2t 2t 2</div>
asdfasd fasd
</div>
</div>
<br>
<br>
<br>
test
</span>
