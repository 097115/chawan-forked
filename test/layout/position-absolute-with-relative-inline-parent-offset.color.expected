
Note: the absolute box is sized differently than boxes in other engines, because
the CSS standard explicitly specifies this as UB and I *really* don't want to
make this even more complex.

I don't care, but feel free to fix it ;)



test test test testadsfasdf
test test [48;2;255;0;0m2t 2tt[49mest
asdfasd fa[48;2;255;0;0m2d    [49m
          [48;2;255;0;0m      [49m


test 
